### ISSUE TITLE
`DfContext` HTML entity decoding is incomplete and introduces platform-specific inconsistencies

### ISSUE THREAD

**Issue by VasilevNStas** [objectionary/lints#1386]
## Problem

`DfContext` (which decodes HTML entities in defect `@context`) decodes incompletely and platform-dependently, so contexts can contain literal `&lt;`/`&quot;` remnants and differ between operating systems.

## Root cause

`src/main/java/org/eolang/lints/DfContext.java:60-64`:

```java
this.context = defect.context()
    .replace("&#10;", System.lineSeparator())
    .replace("&#34;", "\"")
    .replace("&amp;", "&");
```

1. `&amp;` is replaced **last**, so double-encoded input like `&amp;lt;` becomes `&lt;` (still encoded) instead of `<`.
2. The standard entities `&quot;` and `&#39;`/`&apos;` are not handled (only `&#34;`), so a context using `&quot;` keeps the literal `&quot;`.
3. `&#xA;` is replaced with `System.lineSeparator()` (`\r\n` on Windows, `\n` on Unix), while XML normalization always means LF — the resulting context differs across OSes and breaks string comparisons.

## Minimal example

A context value `&amp;lt;o&amp;gt;` should decode to `<o>` but yields `&lt;o&gt;`; a value `&quot;x&quot;` keeps `&quot;x&quot;` verbatim.

## Expected behavior

Decode entities in a single normalized pass:
- replace `&amp;` first (or loop until stable),
- handle `&quot;`, `&#39;`, `&apos;`, `&#10;`/`&#xA;`,
- use a fixed `\n` for line breaks, not `System.lineSeparator()`.

## Related

- `src/test/java/org/eolang/lints/DfContextTest.java` — only covers the happy path; the double-encoding and entity variants are untested.

**Comment by 0crat** [comment]
@vasilevnstas Thanks for reporting a new bug! You've earned +12 points. By reporting bugs, you help our project improve its quality. If you find anything else in the repository that [doesn't look](https://www.yegor256.com/2018/02/06/where-to-find-more-bugs.html) as good as you might expect, [do not hesitate](https://www.yegor256.com/2014/04/13/bugs-are-welcome.html) to [report](https://www.yegor256.com/2018/04/24/right-way-to-report-bugs.html) it. Your running score is [+854](https://www.eolang.org/zerocracy/objectionary-vitals.html); don't forget to check your [Zerocracy account](https://www.zerocracy.com/) too).
